addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -453,6 +462,346 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Should display a list of suggested recipients after opening the composer from 'Send message' button", async function (assert) {
```

Comment:
_"suggest recipient on 'Send message' composer"_

_(it's fine to be broad on feature. This is the 1st test of this feature, and every other tests on this feature has this as pre-requisite)_